src/screens/LessonsList.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import {FlatList, StyleSheet, Text, View, TouchableOpacity} from 'react-native';
+import {DATA_LEVEL_OVERVIEW} from '../consts/lessons';
```

Comment:
space on top.

Reply:
@mark-yacoub  can you elaborate ?

Reply:
leaver space between react libs and our comps